.github/workflows/ci.yml

```diff
@@ -54,7 +54,16 @@ jobs:
       run: clang --version
 
     - name: Run pytests
-      run: python -m pytest apis/python/tests
+      # Setting PYTHONPATH ensures the tests load the in-tree source code unde apis/python/src
+      # instead of copy we `pip install`ed to site-packages above. That's needed for the code
+      # coverage analysis to work.
+      run: PYTHONPATH=$(pwd)/apis/python/src python -m pytest --cov=apis/python/src --cov-report=xml apis/python/tests
+
+    - name: Report coverage to Codecov
+      if: matrix.os == 'ubuntu-22.04' && matrix.python-version == '3.9'
+      uses: codecov/codecov-action@v3
+      with:
+        flags: python
```

Comment:
@eddelbuettel

Reply:
Yay, that should do the trick from what I have seen.